SimTracker/SiPhase2Digitizer/test/Phase2TrackerMonitorDigi.cc

```diff
@@ -117,7 +114,7 @@ void Phase2TrackerMonitorDigi::fillDigiHistos(const edm::Handle<edm::DetSetVecto
     int nclus = 0;
     int width = 0;
```

Comment:
`width` should be initialized to 1; otherwise, there will be a division by zero on line 143. On line 141 do:
```
if (width != 1)
  width++;
```